malaria-ios/ViewControllers/RapidFireViewController.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+class RapidFireViewController: UIViewController {
+  
+  @IBOutlet weak var questionLabel: UILabel!
+  @IBOutlet var answerButtons: [UIButton]!
+  @IBOutlet weak var countDownLabel: UILabel!
+  @IBOutlet weak var scoreLabel: UILabel!
+  
+  @IBInspectable let TimerBlinkAtSecond: Int = 3
+  @IBInspectable let TimerMaxValue: Int = 5
+  
+  var rapidFireGame: RapidFireGame?
+  
+  private var timer: NSTimer?
+  
+  private var currentLevel = 0 {
+    didSet {
+      // Check if we ran out of questions
+      if rapidFireGame!.entries.count == currentLevel {
+        endGame()
+        return
+      }
+      
+      // Set the question
+      questionLabel.text = rapidFireGame?.entries[currentLevel].question
+      
+      // Set the answer buttons
+      let answers = rapidFireGame?.entries[currentLevel].answers
+      for (index, answer) in (answers?.enumerate())! {
+        answerButtons[index].setTitle(answer, forState: .Normal)
+      }
+      
+      // Set timer
+      timer = NSTimer.scheduledTimerWithTimeInterval(1, target: self, selector: #selector(updateTimer), userInfo: nil, repeats: true)
+    }
+  }
+  
+  private var userScore: Int = 0 {
+    didSet {
+      scoreLabel.text = "Score: \(userScore)"
+    }
+  }
+  
+  private var count = 5 {
+    didSet {
+      countDownLabel.text = count != 1 ? "\(count) seconds" : "\(count) second"
+    }
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    // Set background image
+    view.backgroundColor = UIColor(patternImage: UIImage(named: "background")!)
+    
+    // Start game
+    currentLevel = 0
```

Comment:
on viewDidLoad? Is it required to put this here instead of the class field (I will have to recall the life cycle of the view controllers). Or did you intended to put this on the viewWillAppear which makes more sense at this moment for me but I might not be aware of everything.